src/app/contexts/ToggleContext/index.jsx

```diff
@@ -1,66 +1,23 @@
-import React, { createContext, useEffect, useReducer } from 'react';
-import { node, string } from 'prop-types';
-import webLogger from '#lib/logger.web';
-import { toggleReducer, updateToggles } from './reducer';
+import React, { createContext, useReducer } from 'react';
+import { node, string, shape } from 'prop-types';
+import { toggleReducer } from './reducer';
 import defaultToggles from '#lib/config/toggles';
-import constructTogglesEndpoint from './utils/constructTogglesEndpoint';
 
-const logger = webLogger();
 const ToggleContext = createContext({});
 
-const ToggleContextProvider = ({ children, service, origin }) => {
+const ToggleContextProvider = ({ children, remoteToggles }) => {
   const environment = process.env.SIMORGH_APP_ENV || 'local';
-  const simorghToggles = defaultToggles[environment];
+
+  const simorghToggles = {
```

Comment:
As above - do we want to move this merging into the `getRemoteToggles` function and just pass them into here? Maybe as a follow up PR.